custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -15,6 +15,10 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
                 $location.search('location_name', loc.name);
             }
         };
+        vm.selectedLocationsCount = function() {
```

Comment:
this feels comment worthy - like why only subtract 1 if location level is not 4?

Reply:
like I said in the above comment size of selected locations is fixed (5). when select till sector level, entries are (national, state, district, block, sector). When awc is selected its still 5 (replacing national with awc). In html you can see filtering out "all" option. so, in js we are subtracting 1 when awc is not selected.